.gitignore

```diff
@@ -22,6 +22,7 @@ pgfinder/debug_enviroment
 pgfinder/local_test.py
 *~
 \#*
+\#*\#
```

Comment:
Think I might have worked this out, each `EXCLUDE_PATTERN` needs to have `../` prepended since it doesn't automatically take into account the `<MODULE_PATH>`.  Going to give it a whirl :crossed_fingers: